npm link of sdk does not seem to work

I can use this approach in browserify based apps, but not here.

Might be related to: https://github.com/webpack/webpack/issues/811

```
ERROR in ../sdk/release/components/QGISPrint.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/QGISPrint.js 43:14-33

ERROR in ../sdk/release/components/Zoom.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/Zoom.js 43:14-33

ERROR in ../sdk/release/components/Rotate.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/Rotate.js 39:14-33

ERROR in ../sdk/release/components/HomeButton.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/HomeButton.js 45:14-33

ERROR in ../sdk/release/components/Globe.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/Globe.js 51:14-33

ERROR in ../sdk/release/components/LayerList.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/LayerList.js 103:14-33

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 77:14-33

ERROR in ../sdk/release/components/LayerList.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/LayerList.js 83:21-47

ERROR in ../sdk/release/components/LayerList.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/LayerList.js 87:13-31

ERROR in ../sdk/release/components/LayerList.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/LayerList.js 91:21-47

ERROR in ../sdk/release/components/LayerList.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/LayerList.js 95:20-45

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 55:16-38

ERROR in ../sdk/release/components/LayerList.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/LayerList.js 75:22-59

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 63:18-51

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 67:19-53

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 71:25-65

ERROR in ../sdk/release/components/QGISPrint.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/QGISPrint.js 39:18-49

ERROR in ../sdk/release/components/MapPanel.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/MapPanel.js 43:18-49

ERROR in ../sdk/release/components/LayerList.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/LayerList.js 79:18-49

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 39:18-49

ERROR in ../sdk/release/components/Globe.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/Globe.js 43:19-52

ERROR in ../sdk/release/components/LayerList.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/LayerList.js 59:20-54

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 51:21-60

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 59:21-58

ERROR in ../sdk/release/components/LayerList.js
Module not found: Error: Cannot resolve module 'style-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/LayerList.js 133:0-26

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'style-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 91:0-26

ERROR in ../sdk/release/components/InfoPopup.js
Module not found: Error: Cannot resolve module 'style-loader' in /home/bartvde/opengeo/git/sdk/release/components
 @ ../sdk/release/components/InfoPopup.js 93:0-26

ERROR in ../sdk/release/services/MapConfigService.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/services
 @ ../sdk/release/services/MapConfigService.js 35:22-49

ERROR in ../sdk/release/services/MapConfigService.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/services
 @ ../sdk/release/services/MapConfigService.js 31:12-30

ERROR in ../sdk/release/services/MapConfigService.js
Module not found: Error: Cannot resolve module 'babel-loader' in /home/bartvde/opengeo/git/sdk/release/services
 @ ../sdk/release/services/MapConfigService.js 39:18-41

ERROR in ./~/css-loader!../sdk/release/dist/css/components.css
Module not found: Error: Cannot resolve module 'file-loader' in /home/bartvde/opengeo/git/sdk/release/dist/css
 @ ./~/css-loader!../sdk/release/dist/css/components.css 6:20525-20584

ERROR in ./~/css-loader!../sdk/release/dist/css/components.css
Module not found: Error: Cannot resolve module 'file-loader' in /home/bartvde/opengeo/git/sdk/release/dist/css
 @ ./~/css-loader!../sdk/release/dist/css/components.css 6:20746-20799 6:20824-20877

ERROR in ./~/css-loader!../sdk/release/dist/css/components.css
Module not found: Error: Cannot resolve module 'file-loader' in /home/bartvde/opengeo/git/sdk/release/dist/css
 @ ./~/css-loader!../sdk/release/dist/css/components.css 6:20927-20981

ERROR in ./~/css-loader!../sdk/release/dist/css/components.css
Module not found: Error: Cannot resolve module 'file-loader' in /home/bartvde/opengeo/git/sdk/release/dist/css
 @ ./~/css-loader!../sdk/release/dist/css/components.css 6:21011-21064

ERROR in ./~/css-loader!../sdk/release/dist/css/components.css
Module not found: Error: Cannot resolve module 'file-loader' in /home/bartvde/opengeo/git/sdk/release/dist/css
 @ ./~/css-loader!../sdk/release/dist/css/components.css 6:21098-21151
```

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -32,6 +32,9 @@ module.exports = {
   resolve: {
     extensions: ['', '.js', '.jsx']
   },
+  resolveLoader: {
+    root: path.join(__dirname, 'node_modules')
+  },
 	module: {
 		loaders: [
       {

